cloud_governance/policy/aws/empty_roles.py

```diff
@@ -3,6 +3,10 @@
 
 
 class EmptyRoles(NonClusterZombiePolicy):
+    """
+    This class lists and deleted the empty roles ( doesn't have any policies ) after 7 days of empty,
+    send an alert mail to the user after 4 days of empty.
+    """
```

Comment:
This class sends an alert mail for empty role to the user after 4 days and delete after 7 days.